Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+- This approach also generically works for any IPv4 address returned/embedded
+  from the AAAA record.
+
+This approach has the downside that:
+
+- Involved to set up. The Kubernetes cluster needs a special IPAM where nodes or Pods
+  get a secondary IPv6 address in the form of ``[64:ff9b::IPv4_NP]``.
+  The CNI needs to install a routing rule into Pods to select ``[64:ff9b::IPv4_NP]`` as
+  a source address for all ``64:ff9b::/96``-destined traffic.
+- Due to the ``[64:ff9b::IPv4_NP]`` format, the benefits of a highly scalable IPv6 IPAM
+  may be limited since routable IPv4 CIDRs are needed. This may be less of a problem if
+  not all Pods are in need for connecting to IPv4.
+- A next hop route is needed on each cluster node to direct replies for all
+  ``64:ff9b::/96``-destined traffic towards the gateway nodes for translation.
+- The cluster needs to be configured with DNS64 in order to return ``64:ff9b::``-prefixed
+  AAAA records.
+
+Cilium for Kubernetes currently does not support such an IPAM mode where Pods
+receive addressing from two different IPAM pools, but this is something that
+is planned to resolve in next Cilium releases so that for Kubernetes this
+scenario will be supported, too.
+
+Tuning Considerations
+---------------------
+
+For optimal performance, we recommend the following:
+
+    * Enable native XDP for the gateway: ``--bpf-lb-acceleration=native``
+      See also which devices :ref:`support XDP natively<XDP Acceleration>`.
+    * Enable faster access to clock for the gateway: ``--enable-bpf-clock-probe=true``
+    * Have a recent, up-to-date kernel and consider building it
+      with ``CONFIG_PREEMPT_NONE=y`` for server-type workloads.
+    * Stop ``irqbalance`` and pin the NIC's RX/TX IRQs to CPUs with
+      a 1:1 affinity mapping for maximum isolation/siloing. In case
+      of multi-socket nodes, it is advisable to ensure to pin the IRQs
+      to the NUMA node where the NIC slot is connected to in order to
+      lower inter-node transfers and to use only local node memory.
+    * If necessary, set the CPU governor to performance profile.
+    * Consider larger map sizes than the default, depending on your
+      scale needs. Please refer to the :ref:`bpf_map_limitations` guide.
+
+Some of these are elaborated further on in our general :ref:`performance_tuning`
+document.
+
+Limitations
+===========
+
+The following limitations below apply at this point in time:
+
+    * Cilium's eBPF load-balancer does not yet support XDP multi-buffer mode
+      for supporting 9k MTUs.
+    * Backend weights is currently only supported under Maglev and not yet
+      Random mode.
+    * Upon Cilium restart the backend weights are preserved as-is, however, the
+      displayed weights value in the ``cilium service list`` json/yaml dump is
+      currently not preserved and will display ``1``. See further details
+      in `GH-18306 <https://github.com/cilium/cilium/pull/18306>`_. This does
```

Comment:
```suggestion
      in :gh-issue:`18306`. This does
```

Reply:
^ There's probably more cases of this, we use the formatter to get consistent output when referring to github issues across the docs.